### PR TITLE
Fail if mirror issues

### PR DESCRIPTION
**Pull request by betoredhat**

### CODE REVIEW

File: roles/mirror_catalog/README.md [hunk 1]
```diff
@@ -4,13 +4,14 @@ Mirrors a catalog and its related images. Produces a file that can be used to se
...
 
-Name            | Required | Default | Description
-----------------|----------| --------|-------------
-mc_oc_tool_path | Yes      |         | The path to the oc<sup>1</sup> binary, e.g. '/path/to/oc'
-mc_catalog      | Yes      |         | The Fully Qualified Artifact Reference, e.g. 'example.com/namespace/web:v1.0'
-mc_registry     | Yes      | ""      | The registry where the catalog will be mirrored, e.g. 'registry.example.com' or 'reg.example.com:4443'
-mc_pullsecret   | No       |         | The credential file to pull and/or push the images, e.g. '/path/to/pullsecret.json'
-mc_is_type      | No       | icsp    | The type of image source to use, choose between icsp (imageContentsourcePolicy) (default) or idms (imageDigestMirrorSet).
+Name            | Required | Type     | Default | Description
+----------------|----------| ---------|-------------
```
Comment (tonyskapunk) [inline roles/mirror_catalog/README.md:8]:
```suggestion
--------------- | -------- | -------- | ------------
```

**Comment by betoredhat** [comment]
This has been validated in the jobs above, in the 4.15 it fails because there are images missing during the mirroring. In 4.14 the issue is not related to this change but the mirroring passed.

**Review by manurodriguez — APPROVED** [review]
LGTM

File: roles/mirror_catalog/README.md [hunk 2]
```diff
@@ -4,13 +4,14 @@ Mirrors a catalog and its related images. Produces a file that can be used to se
...
 
-Name            | Required | Default | Description
-----------------|----------| --------|-------------
-mc_oc_tool_path | Yes      |         | The path to the oc<sup>1</sup> binary, e.g. '/path/to/oc'
-mc_catalog      | Yes      |         | The Fully Qualified Artifact Reference, e.g. 'example.com/namespace/web:v1.0'
-mc_registry     | Yes      | ""      | The registry where the catalog will be mirrored, e.g. 'registry.example.com' or 'reg.example.com:4443'
-mc_pullsecret   | No       |         | The credential file to pull and/or push the images, e.g. '/path/to/pullsecret.json'
-mc_is_type      | No       | icsp    | The type of image source to use, choose between icsp (imageContentsourcePolicy) (default) or idms (imageDigestMirrorSet).
+Name            | Required | Type     | Default | Description
+--------------- | -------- | -------- | ------------
```
Comment (tonyskapunk) [inline roles/mirror_catalog/README.md:8]: Sorry forgot there is a new field
```suggestion
--------------- | -------- | -------- | ------- | -----------
```

**Pull request merged**